lms/djangoapps/grades/tasks.py

```diff
@@ -33,6 +33,7 @@
 log = getLogger(__name__)
 
 KNOWN_RETRY_ERRORS = (DatabaseError, ValidationError)  # Errors we expect occasionally, should be resolved on retry
+RECALCULATE_GRADE_DELAY = 2  # in seconds, to prevent excessive _has_db_updated failures. See TNL-6424.
```

Comment:
@jibsheet - this is the delay we're adding to to the recalculate grades task to alleviate the "task ran before the requesting problem_check transaction was committed" errors from loadtest last week. Any devops concerns about needing to redefine this value on-the-fly?